internal/types/policy.go

```diff
@@ -0,0 +1,40 @@
+package types
```

Comment:
We shoudn't re-introduce this - the policy type is public under the top-level types package now - please merge any relevant changes from the new code there so it can be imported by the plugin package.